src/VisualStudio/Core/Def/Extensions/BulkObservableCollectionExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.VisualStudio.Language.Intellisense;
+
+namespace Microsoft.VisualStudio.LanguageServices.Extensions;
+
+internal static class BulkObservableCollectionExtensions
+{
+    public static BulkOperationDisposable<T> GetBulkOperation<T>(this BulkObservableCollection<T> collection)
```

Comment:
helper so we can just do `using (obColl.GetBulkOperation())` instead of a try/finally.